source/content/terminus/install.md

```diff
@@ -171,3 +171,14 @@ On a subsystem with Ubuntu 18.04, since Curl is already installed, the commands
 sudo apt update && sudo apt upgrade -y
 sudo apt install php7.2-cli
 ```
+#### POSIX support required
+
+POSIX extension support is required to run Terminus. To verify that you have the POSIX extensions available in your PHP, try:
+
+```bash{promptUser:user}
+php -r 'var_export(function_exists("posix_isatty"));'
+```
+
+If this returns false, try installing the [POSIX extensions](https://www.php.net/manual/en/posix.installation.php). 
+
+Windows' lack of POSIX extension support causes errors, including no response to remote Drush commands. Some have found  [installing `php73-process.x86_64`](https://github.com/pantheon-systems/terminus/issues/1924#issuecomment-629440151) provides the needed POSIX support. 
```

Comment:
This comment is for Linux users, not for Windows users. I think that there isn't an option to install POSIX on Windows.

The workaround for Windows is to run with `--no-interaction`. This worked for the Linux user without POSIX installed, but I'm not sure if it is effective for Windows.